watchify/watchify.d.ts

```diff
@@ -0,0 +1,51 @@
+// Type definitions for watchify v3.7.0
+// Project: https://github.com/substack/watchify
+// Definitions by: TeamworkGuy2 <https://github.com/TeamworkGuy2>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../browserify/browserify.d.ts" />
+
+declare module 'watchify' {
+
+    var Watchify: Watchify.Constructor;
+
+    /** Watch mode for browserify builds.
+     * Update any source file and your browserify bundle will be recompiled on the spot
+     */
+    module Watchify {
```

Comment:
namespaces can not be used with a string (module identifier), these are modules. 

In TS 1.0 `module` meant both, external module (something you get by calling `require`), and an internal module (a named object literal to avoid namespace pollution). this was confusing.  so TS 1.6 added a new keyword `namespace` to be used for internal modules.

here are more docs about this: http://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html